plugins/repository-hdfs/build.gradle

```diff
@@ -42,7 +42,7 @@ dependencies {
   compile "org.apache.hadoop:hadoop-auth:${versions.hadoop2}"
   compile "org.apache.hadoop:hadoop-hdfs:${versions.hadoop2}"
   compile 'org.apache.htrace:htrace-core:3.1.0-incubating'
-  compile 'com.google.guava:guava:16.0.1'
+  compile "com.google.guava:guava:${versions.guava}"
```

Comment:
I'm surprised that we can get away with 16.0.1 here as Hadoop [declares a dependency on 11.0.2](https://github.com/apache/hadoop/blob/159d6c56e7f3aa3ebe45750cf88735287f047b42/hadoop-project/pom.xml#L509-L513). I've personally seen problems with using a different Guava version than the version that Hadoop expects. I'd prefer that we not make this change.